clients/src/main/java/org/apache/kafka/clients/consumer/KafkaConsumer.java

```diff
@@ -1782,6 +1782,13 @@ public void close() {
      * timeout. If the consumer is unable to complete offset commits and gracefully leave the group
      * before the timeout expires, the consumer is force closed. Note that {@link #wakeup()} cannot be
      * used to interrupt close.
+     * <p>
+     * The actual maximum wait time is bounded by the {@link ConsumerConfig#REQUEST_TIMEOUT_MS_CONFIG} setting, which
+     * only applies to operations performed with the coordinator (coordinator-related requests and
```

Comment:
Should we say this instead? (could be to the coordinator or the leader)
```suggestion
     * only applies to operations performed with the broker (coordinator-related requests and
```